doc/source/user_guide/parameters.rst

```diff
@@ -42,3 +42,148 @@ ROUT``, you can access it with:
 
 For a full listing of the methods and attributes available to the
 ``Parameters`` class, please reference the :ref:`ref_parameters_api`.
+
+
+Especially Named Parameters
+===========================
+
+Leading Underscored Parameters
+------------------------------
+
+The parameters starting with underscore (``'_'``) are reserved parameters
+for MAPDL macros and routines, therefore its use is discouraged, and
+in PyMAPDL you cannot set them by default.
+
+If you need to set one of this parameters, you can use
+:attr:`Mapdl._run <ansys.mapdl.core.Mapdl._run>`
+to avoid PyMAPDL parameter name checks. For example
+
+
+.. code:: python
+
+   >>> mapdl._run('_parameter=123')
+   'PARAMETER _PARAMETER =     123.00000000'
+
+By default, this type of parameters cannot be seen when issuing
+:attr:`Mapdl.parameters <ansys.mapdl.core.Mapdl.parameters>`.
+However, you can change this by setting
+:attr:`Mapdl.parameters.show_leading_underscore_parameters 
+<ansys.mapdl.core.Mapdl.parameters.show_leading_underscore_parameters>`
+equal to ``True``.
+For example:
+
+
+.. code:: python
+
+   >>> mapdl.parameters.show_leading_underscore_parameters=True
+   >>> mapdl.parameters
+   MAPDL Parameters
+   ----------------
+   PORT                             : 50053.0
+   _RETURN                          : 0.0
+   _STATUS                          : 0.0
+   _UIQR                            : 17.0
+
+
+Trailing Underscored Parameters
+-------------------------------
+
+The parameters ending underscore are recommend for user routines
+and macros.
+You can set this type of parameters in PyMAPDL, but by default,
+they cannot be seen in
+:attr:`Mapdl.parameters <ansys.mapdl.core.Mapdl.parameters>`, unless
+:attr:`Mapdl.parameters.show_trailing_underscore_parameters 
+<ansys.mapdl.core.Mapdl.parameters.show_trailing_underscore_parameters>`
+is set to ``True``.
+
+
+.. code:: python
+
+   >>> mapdl.parameters['param_'] = 1.0
+   >>> mapdl.parameters
+   MAPDL Parameters
+   ----------------
+   >>> mapdl.parameters.show_trailing_underscore_parameters=True
+   >>> mapdl.parameters
+   MAPDL Parameters
+   ----------------
+   PARAM_                           : 1.0
+
+
+Parameters with Leading and Trailing Underscore
+-----------------------------------------------
+
+These are an especial type of parameters. They **CANNOT** be seen
```

Comment:
These are a special type of parameter. They **CANNOT** be seen in :attr:`Mapdl.parameters <ansys.mapdl.core.Mapdl.parameters>` under any circumstances, and therefore their use is not recommended.